toucan_connectors/mysql/mysql_connector.py

```diff
@@ -99,9 +99,7 @@ class Config:
         keep_untouched = (cached_property_with_ttl,)
 
     def _list_db_names(self) -> list[str]:
-        connection = pymysql.connect(
-            **self.get_connection_params(cursorclass=None, database='mysql')
-        )
+        connection = pymysql.connect(**self.get_connection_params(cursorclass=None, database=None))
```

Comment:
It could be a good idea to do the same thing as it's done on Postgres with a default database parameter, what do you think? 🤔 

https://github.com/ToucanToco/toucan-connectors/blob/master/toucan_connectors/postgres/postgresql_connector.py#L227

Reply:
Maybe in another PR, but we'd have to dig first, as Postgres and MySQL have different behaviours regarding metadata. Plus I don't want to introduce a side effect on a connector in a fix targetting MySQL

Reply:
Good point ! 👍🏽